solution/backend/regulations/views/homepage.py

```diff
@@ -23,7 +23,7 @@ def get_context_data(self, **kwargs):
         today = date.today()
         parts = Part.objects.effective(today)
         resources_config = ResourcesConfiguration.objects.first()
-        fr_docs_category_name = resources_config.fr_doc_category.name
+        fr_docs_category_name = resources_config.fr_doc_category.name if resources_config.fr_doc_category else ""
```

Comment:
minor fix to ensure we're not attempting to access a property of a NoneType, which throws an error that prevents loading the site.